src/index.html

```diff
@@ -17,11 +17,449 @@
-      href="styles/index.scss"
+      href="styles/main.scss"
     />
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="menu">
+        <ul class="menu__list">
+          <li class="menu__item">
+            <a
+              class="is-active menu__link"
+              href="apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              data-qa="nav-hover"
+              href="laptops"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="menu__item">
+            <a
+              class="menu__link"
+              href="video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <div class="card__photo"></div>
+
+        <a
+          href="#"
+          class="card__name"
+        >
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </a>
+
+        <span class="card__product">Product code: 195434</span>
+
+        <div class="card__reviews">
+          <div class="stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <a
+            href="#"
+            class="reviews"
+          >
+            Reviews: 5
+          </a>
+        </div>
+
+        <div class="price">
+          <span class="price__name">Price:</span>
+          <span class="price__number">$2,199</span>
+        </div>
+
+        <button
+          class="card__button"
+          data-qa="card-hover"
```

Comment:
The 'button' element should have a 'type' attribute. The most common values are 'submit', 'reset', and 'button'. If not specified, the default value is 'submit'.